homework1.html

```diff
@@ -0,0 +1,60 @@
+<html>
+<head>
+    <title>Home Work1</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+</head>
+
+<body>
+    <div class="header-content1">
+        <div class="header">
+            <div class="menu-item ">
+                <span class="Home"> Home    
+                </span>
+            </div>
+
+            <div class="menu-item">
+                <span class="About">About   
+                </span>
+            </div>
+                    
+            <div class="menu-item">
+                <span class="Contact">Contact  
+                </span>
+            </div>
+        </div>
+    </div>
+
+    <div class="main">
+        <section class="card">
+            <div>
+                <h5>Group Tour</h5>
+                <img src="Pics/Group.jpg" class="img-responsive img-rounded" alt="Cinque Terre">
+                </img>
+                <p> I missed Group Tour during this period</p>
+            </div>
+        </section>  
+        <section class="card">
+            <div>
+                <h5>Walking on Beach </h5>
+                <img src="Pics/Walking on Beach.jpg" class="img-responsive img-rounded" alt="Cinque Terre">
+                </img>
+                <p> I missed Walking on Beach during this period</p>
+            </div>
+        </section>
+        <section class="card">
+            <div>
+                <h5>Travel by Air</h5>
+                <img src="Pics/Travel.jpg" class="img-responsive img-rounded" alt="Cinque Terre">
+                </img>
+                <p> I missed Travel by Air at this Pandemic period</p>
+             </div>
+        </section>
+    </div>
+    
+    <footer class="footer" >
```

Comment:
You don't have this class specified in css file, please add it there so your footer is styled.